webcompat/views.py

```diff
@@ -155,8 +150,18 @@ def show_issues():
     return render_template('list-issue.html', categories=categories)
 
 
-@app.route('/issues/new', methods=['POST'])
+@app.route('/issues/new', methods=['GET', 'POST'])
 def create_issue():
+    """Creates a new issue
+
+    GET will return an HTML response for reporting issues
+    POST will create a new issue
+    """
```

Comment:
I have the feeling we could improve the way we handle the code here. But probably for another issue, you might want to open.
Right now we have

```python
if GET:
    this_code()
    return blah_get
# here the http post code (with logging and spam)
return blah_post
```

Maybe something along

```python
# common_tasks
# ua_header, etc.
if GET:
    this_code()
    return blah_get
if POST:
   # making functions for different features
    spam_protect()
    log_this()
    return blah_post
```

BUT Let's not block the progress of this PR. so +1 with the issue created.